core/src/com/unciv/logic/civilization/VictoryManager.kt

```diff
@@ -37,7 +37,7 @@ class VictoryManager {
             && civInfo.policies.adoptedPolicies.count{it.endsWith("Complete")} > 4
 
     fun hasWonDominationVictory() = civInfo.gameInfo.gameParameters.victoryTypes.contains(VictoryType.Domination)
-            && civInfo.gameInfo.civilizations.all { it==civInfo || it.isDefeated() || !it.isMajorCiv() }
+            && civInfo.gameInfo.civilizations.all { it==civInfo || it.isDefeated() || !it.isMajorCiv() || it.isSpectator() }
```

Comment:
Should not be necessary, since !it.isMajorCiv() also includes it.isSpectator()

Reply:
Right. I forgot that added this before excluded Spectators from MajorCivs.